### PR TITLE
Switch to more recent DUTI fork

### PR DESCRIPTION
**Pull request by holdhack**
https://github.com/moretension/duti hasn't been updated in three years. Not only is https://github.com/russellhancox/duti more active, switching to this fork also allows us to drop the autoconf dependency and the custom patch for 10.11+ :)

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
  Warning 1 = 'Stable: Use versioned rather than branch tarballs for stable checksums' - I have no control over the repo, and the versioned branches are all out of date.
  Warning 2 = 'Commented-out dep "cmake" => :build' - No idea why this warning is being thrown out. I removed Cmake from my system and all the install and all tests go just fine.

-----

### CODE REVIEW

File: Formula/duti.rb
Comment (neutric) [inline Formula/duti.rb]: Why not utilize the Makefile: `system "make", "install", "BINDIR=#{bin}", "MANDIR=#{man1}"`?

File: Formula/duti.rb
Comment (neutric) [inline Formula/duti.rb]: We need a tagged version to avoid breakage down the line. Please submit an issue upstream asking them to tag a new version.

File: Formula/duti.rb
Comment (neutric) [inline Formula/duti.rb]: Please remove.

**Review by neutric — CHANGES_REQUESTED** [review]
I reviewed your changes but haven't been maintainer long enough to be able to comment on switching forks. You might want to wait with changes until you hear from more experienced maintainers. Pinging the ever so helpful @ilovezfs. :)

**Comment by ilovezfs** [comment]
It's unusual to switch to a fork unless the fork has effectively become upstream by becoming significantly more notable on GitHub than the original repository, as measured by watchers, stars, issue counts, PR counts, commit frequency, etc. In this case, the fork has 1 watcher, 0 stars, and issues disabled, so it's not really in the cards at this point.

@jorvi That said, if there are useful commits in the fork, I'd encourage you to try to get them upstreamed into the main repository, even if it currently appears to be dormant.

**Comment by MikeMcQuaid** [comment]
> @jorvi That said, if there are useful commits in the fork, I'd encourage you to try to get them upstreamed into the main repository, even if it currently appears to be dormant.

@jorvi Alternatively, if you can get the existing main repository to bless the fork as the new, canonical source in the README: that works too.

**Comment by neutric** [comment]
Thanks for your feedback, guys!

**Comment by jorvi** [comment]
Thanks! @MikeMcQuaid there's actually two forks that are interesting, the one I mentioned and https://github.com/nivekkagicom/duti, which has some extra functionality. Would it be cool with you/Homebrew if I merged the two, made a 1.7.1 branch, and got blessing from the main repository?
I use DUTI in my dotfiles so I'd be happy to maintain both DUTI and the Homebrew formula.

File: Formula/duti.rb
Comment (jorvi) [inline Formula/duti.rb]: That doesn't work, you simply end up with an empty directory in the Cellar

**Comment by jorvi** [comment]
@neutric @MikeMcQuaid 

Sorry for the late response. I've tried to contact the original repo in several ways, but as said in #2840, Upstream is dead, the guy hasn't been on GitHub in 3 years neither has he updated the project. The fork I originally pointed to also does not respond.
Since I use this tool myself, I'm happy to maintain both the binary and the formula.
I've updated the formula with the correct SHA256. I've made a new version on the repo, as requested.
Again, as said in #2840, this tool sees a reasonable amount of use, so it'd be kind of sad to put it down..

**Comment by MikeMcQuaid** [comment]
> there is an active fork now, so it'd be kind of sad to put it down..

We need to have the active fork be blessed by upstream, sorry.

**Comment by jorvi** [comment]
@MikeMcQuaid 
But there is no way to do that if the owner of upstream **hasn't been on GitHub for over 3 years, nor does he have any point of contact**. You guys already removed DUTI, and here presents itself an option to keep a moderately popular tool active. Please see the reason in this.. or we can roll with the extremely kafkaesque solution of me renaming the tool to `UTIlizer` or `UTID` or whatever and asking for this new tool to be included on Homebrew.

**Comment by MikeMcQuaid** [comment]
> extremely kafkaesque solution of me renaming the tool to UTIlizer or UTID

When people `brew install duti` they expect the canonical version be installed. If you want a long-running fork that doesn't have upstream blessing: it needs a new name.

**Pull request closed without merging**
